README.md

```diff
@@ -1,120 +1,26 @@
-<!-- Update this title with a descriptive name. Use sentence case. -->
-# Terraform modules template project
+# Sample Deployable Architectures
 
-<!--
-Update status and "latest release" badges:
-  1. For the status options, see https://terraform-ibm-modules.github.io/documentation/#/badge-status
-  2. Update the "latest release" badge to point to the correct module's repo. Replace "terraform-ibm-module-template" in two places.
--->
 [![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
-[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-module-template?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-module-template/releases/latest)
+[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/sample-deployable-architectures?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/sample-deployable-architectures/releases/latest)
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-<!--
-Add a description of modules in this repo.
-Expand on the repo short description in the .github/settings.yml file.
+This repository contains the following sample deployable architectures:
+- [Terraform based fullstack sample DA - COS bucket replication](./solutions/tf-fullstack-da)
+- [Terraform based extension sample DA](./solutions/tf-extension-da) [coming soon]
+- [IBM Cloud Projects Stack sample DA](./solutions/projects-stack-da) [coming soon]
```

Comment:
```suggestion
- [Terraform-based fullstack sample deployable architecture - COS bucket replication](./solutions/tf-fullstack-da)
- [Terraform-based extension sample deployable architecture](./solutions/tf-extension-da) [coming soon]
- [IBM Cloud Projects Stack sample deployable architecture](./solutions/projects-stack-da) [coming soon]
```

Reply:
We want to spell out deployable architecture when possible because "DA" isn't an approved acronym